src/sass/parse.js

```diff
@@ -4149,6 +4181,28 @@ function checkSC(i) {
 }
 
 /**
+ * Get node with spaces newlines and comments
+ * @return {Array} Array containing nodes with spaces (if there are any)
```

Comment:
Please replace the return with `@return {!Node}`? There are outdated comments in the code, I'll clean them up after merging you PR.